package.json

```diff
@@ -17,7 +17,7 @@
     "knip-ci": "knip --no-exit-code --reporter json --config .github/knip.ts",
     "prepare": "husky install",
     "test": "jest --setupFiles dotenv/config --coverage",
-    "worker": "wrangler dev --env dev --port 4000"
+    "worker": "wrangler dev --env dev --port 5000"
```

Comment:
Is there a specific reason that you change this? The template uses `:4000` so all other plugins also, it's tedious having to change it back for no reason

Reply:
The plugin-template comes with `:4000` as standard so it's habitual for plugin devs to copy paste that url into `-config.yml` and expecting it to work thinking something is broken for it to be the dev port lmao.

Ideally don't commit it if you want to use another port, like running multiple plugins locally for example, always change it back to the template port if you can remember to.